sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -390,8 +411,11 @@ public void populateDisplayData(DisplayData.Builder builder) {
       super.populateDisplayData(builder);
       getBigtableConfig().populateDisplayData(builder);
 
-      builder.addIfNotDefault(
-          DisplayData.item("keyRange", getKeyRange().toString()), ByteKeyRange.ALL_KEYS.toString());
+      List<ByteKeyRange> keyRanges = getKeyRanges();
+      for (int i = 0; i < keyRanges.size(); i++) {
+        builder.addIfNotDefault(DisplayData.item("keyRange " + i, keyRanges.get(i).toString()),
```

Comment:
Might this be used with a very large number of key ranges? We probably don't want an excessive number of DisplayData items.

Reply:
Good point, added a limit of 5.